extensions/wikia/Captcha/Captcha.setup.php

```diff
@@ -0,0 +1,177 @@
+<?php
+
+/**
+ * @file
+ * @ingroup Extensions
+ */
+
+if ( !defined( 'MEDIAWIKI' ) ) {
```

Comment:
Do we need to do this check here? I feel like 'MEDIAWIKI' should always be defined, and if someone is trying to run this by itself they have a whole 'nother set of problems.